guides/source/active_storage_overview.md

```diff
@@ -32,6 +32,26 @@ Using Active Storage, an application can transform image uploads or generate ima
 representations of non-image uploads like PDFs and videos, and extract metadata from 
 arbitrary files.
 
+### Requirements
+
+Various features of Active Storage depend on third party applications which Rails 
```

Comment:
What about "software", since that is the term used in the warning further below?

```suggestion
Various features of Active Storage depend on third party software which Rails 
```